app/controllers/pictures_controller.rb

```diff
@@ -24,7 +24,8 @@ def edit
   # POST /pictures
   # POST /pictures.json
   def create
-    @picture = Picture.new(picture_params)
+    @picture = Picture.create(params[:id])
+    #@picture = Picture.create(picture_params)    
```

Comment:
I don't think I am passing the params correctly to the create(). It is not saving the image any more.